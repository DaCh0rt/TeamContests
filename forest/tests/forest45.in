111 982
2230 -489
-2001 8504
7138 -9520
-8521 -2711
4769 -675
-815 -3581
336 -379
-6953 5320
8588 6479
-3081 1206
-9718 -3227
4727 4215
9722 -2475
8567 5904
-688 -9570
-1695 9727
-3192 -8814
-5262 -8005
-6240 6465
-5697 -9686
8769 5936
842 -9329
8447 3910
9366 6749
-1609 7895
-427 -6885
7632 5452
9796 -6613
4750 4572
-9980 -7991
-807 -292
-6433 -5222
-6280 6485
-1673 7032
-2378 -1964
-122 -6947
-636 -5363
-2154 2756
9313 7581
2309 -7718
1242 -2556
9544 -1001
8120 4685
-6095 -6794
2473 -2166
-6895 -201
-7345 -7607
-7359 1317
7534 9244
3849 3916
-7882 -9189
-9372 -457
1586 -5851
-7121 -1817
-9217 -2203
-3419 5258
-2482 -644
6171 6384
-673 8620
-2130 3229
-8555 -2581
-1809 -22
6352 1516
7142 -3881
4364 -3968
-7804 7670
-4099 -370
-9909 -3026
-847 -1687
-2016 8447
1238 -5999
8273 -3392
-1809 2271
-7826 -1660
6138 -446
-5846 3094
7911 -2822
-5410 1570
-6524 4994
-4739 1472
6443 -578
3338 9442
7900 -6797
1314 -372
-9110 8106
399 4831
3559 -7428
-2805 5683
-1607 -5101
9007 4373
-9363 2796
-8678 480
5830 -8517
-2716 5423
-427 -6231
-9513 5313
8678 4019
9610 5100
-215 5122
7203 377
6199 -7426
-2451 9450
-7277 -7186
-4356 -9796
7044 -20
8373 3649
-6602 -281
-8302 151
2309 372
-5387 3156
-9886 4608
