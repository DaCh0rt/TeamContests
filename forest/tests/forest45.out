77271.22 432145890.07
